lib/actions-oclif/env/add.ts

```diff
@@ -171,31 +190,40 @@ async function setServiceVars(
 	options: FlagsDef,
 ) {
 	if (options.application) {
-		const serviceId = await getServiceIdForApp(
-			sdk,
-			options.application,
-			options.service!,
-		);
-		await sdk.models.service.var.set(serviceId, params.name, params.value!);
-	} else {
+		for (const app of options.application.split(',')) {
+			for (const service of options.service!.split(',')) {
+				try {
+					const serviceId = await getServiceIdForApp(sdk, app, service);
+					await sdk.models.service.var.set(
+						serviceId,
+						params.name,
+						params.value!,
+					);
+				} catch (err) {
+					console.error(`${err.message}, application: ${app}`);
+					process.exitCode = 1;
+				}
+			}
+		}
+	} else if (options.device) {
 		const { getDeviceAndAppFromUUID } = await import('../../utils/cloud');
-		const [device, app] = await getDeviceAndAppFromUUID(
-			sdk,
-			options.device!,
-			['id'],
-			['app_name'],
-		);
-		const serviceId = await getServiceIdForApp(
-			sdk,
-			app.app_name,
-			options.service!,
-		);
-		await sdk.models.device.serviceVar.set(
-			device.id,
-			serviceId,
-			params.name,
-			params.value!,
-		);
+		for (const uuid of options.device.split(',')) {
+			for (const service of options.service!.split(',')) {
+				const [device, app] = await getDeviceAndAppFromUUID(
```

Comment:
It looks like this call can be moved to the outer loop, it doesn't need to be re-executed for each service:
```
for (const uuid of options.device.split(',')) {

    const [device, app] = await getDeviceAndAppFromUUID(...);

    for (const service of options.service!.split(',')) {
```

Also, the other loops elsewhere in this PR have try / catch / process.exitCode blocks, but these loops here seem to be missing it.